src/common-utils/install.sh

```diff
@@ -449,15 +449,15 @@ echo -e "\
     ZSH_ALREADY_INSTALLED=${ZSH_ALREADY_INSTALLED}" > "${MARKER_FILE}"
 
 # Display a notice on conda when not running in GitHub Codespaces
-cat << 'EOF' > ${DEV_CONTAINERS_DIR}/conda-notice.txt
+cat << 'EOF' > /usr/local/etc/vscode-dev-containers/conda-notice.txt
```

Comment:
Can this config live in the `conda`/`anaconda` features, that way it's only added when you include the `conda`/`anaconda` features?

Reply:
That makes sense. Moved it.